checker.h

```diff
@@ -0,0 +1,27 @@
+
+#ifndef CHECKER_H
+#define CHECKER_H
+
+#define TEMPERATURE_MIN 0.0
+#define TEMPERATURE_MAX 45.0
+#define IS_TEMPERATURE_NOT_IN_LIMIT(currentValue) IsNotWithinLimit(currentValue, TEMPERATURE_MIN, TEMPERATURE_MAX)

```

Comment:
Why not pass the name of the parameter as well? Like "Temperature"
Then the function can print this parameter, followed by 'not in range'.
Then you don't need three functions, you will only need one. Also, it avoids adding another function for every additional parameter requested.

Reply:
yes, added